README.md

```diff
@@ -21,9 +21,10 @@ write applications using a single API call to either PyQt or PySide.
 It provides support for PyQt5, PyQt6, PySide6, PySide2 using the Qt5 layout
 (where the QtGui module has been split into QtGui and QtWidgets).
 
-Basically, you can write your code as if you were using PySide2
-but import Qt modules from `qtpy` instead of `PySide2` (or `PyQt5`)
+Basically, you can write your code as if you were using PyQt or PySide directly,
+but import Qt modules from `qtpy` instead of `PyQt5`, PyQt6`, `PySide2`, or `PySide6`.
```

Comment:
```suggestion
but import Qt modules from `qtpy` instead of `PyQt5`, `PyQt6`, `PySide2`, or `PySide6`.
```

Fix formatting.

Reply:
Sorry, that was my mistake in my suggestion